pkg/pillar/cmd/domainmgr/containerd.go

```diff
@@ -199,13 +199,11 @@ func ctrCreate(containerID string, ctrdImage containerd.Image) error {
 }
 
 // ctrPrepare prepare an existing container
-func ctrPrepare(ociFilename string, envVars map[string]string, noOfDisks int) (string, error) {
-	containerID := uuid.NewV4().String()
-
+func ctrPrepare(containerPath string, ociFilename string, envVars map[string]string, noOfDisks int) error {
```

Comment:
I am not sure I get this part (or if I get it, agree with it).

Beforehand, we passed `ctrPrepare` the `ociFilename`, i.e. the path to the oci tar file (which I am working to change to just an image root), and it would create the ID, create the container, get the path, etc. domainmgr was ignorant of all of the mechanics underlying containers and paths, just passed it a reference to an image and got back an ID.

This appears to change it by actually passing in the path to the container, making domainmgr aware of (and responsible for) the underlying path? 

I don't mind having domainmgr create the container name (UUID) - this actually makes more sense than having it done inside `ctrPrepare`, as it can focus on building a container independent of name - but why pass it the path? Isn't that a big purpose of using containerd? To leverage its path creation and snapshotting, etc?